src/com/visualization/piechart/piechart.less

```diff
@@ -21,6 +21,7 @@
 
         & .piechart-svg {
```

Comment:
This is then a direct decedent of the `& > .-chart`, and best I can tell it will be the only SVG child of it, so you can omit the class and go with `& > svg {`